python/state_space_estimation/estimation.py

```diff
@@ -2,14 +2,14 @@
 import pandas as pd
 import math
 from tqdm import tqdm
-from itertools import chain, combinations, product
+from itertools import chain, combinations
 from joblib import Parallel, delayed
 from multiprocessing import cpu_count
 from .roles import roles
-from .dag import dag

```

Comment:
Is `dag` used here at all?

Reply:
I think maybe it did earlier, but I forgot to remove it. `dag` isn't used for model fitting, only after the fact for plotting the irfs, etc.